### PR TITLE
Add `.findAll(type)` the store class

### PR DESCRIPTION
**Pull request by jprincipe**

### CODE REVIEW

**Comment by beauby** [comment]
Thanks for the PR! I'm currently afk, but will review it later tonight.

-- 
Lucas Hosseini
lucas.hosseini@gmail.com

File: README.md [hunk 1]
```diff
@@ -46,6 +46,10 @@ Just call the `.find(type, id)` method of your store.
 ```javascript
 var article = store.find('article', 123);
 ```
+or call the `.find(type)` method of your store to get all the models of that type.
```
Comment (beauby) [inline README.md]: Here you refer to `.find(type)`, but you probably mean `.findAll(type)`, right?

File: DOCUMENTATION.md [hunk 2]
```diff
@@ -75,6 +75,18 @@ Retrieve a model by type and id. Constant-time lookup.
 
 * **object** The corresponding model if present, and null otherwise.
 
+## find(type)
```
Comment (beauby) [inline DOCUMENTATION.md]: This should probably be `findAll(type)` as well.

File: test/store.js [hunk 3]
```diff
@@ -250,6 +256,42 @@ describe('JsonApiDataStore', function() {
       var article = store.find('article', 9999);
       expect(article).to.eq(null);
     });
+
+    it('should not find a non-existing model type', function() {
```
Comment (beauby) [inline test/store.js]: :+1:

**Comment by beauby** [comment]
LGTM except for the `find`/`findAll` mix up in the docs!

**Comment by jprincipe** [comment]
@beauby updated docementation

**Comment by jprincipe** [comment]
@beauby I created another branch to convert the library to ES6.  I am using babel to generate the dist files so they are converted to ES5, but should make the source code cleaner.  Any interest?

https://github.com/jprincipe/jsonapi-datastore/tree/es6

**Comment by beauby** [comment]
Looks good! Could you squash the commits before I merge?
Yeah I'm definitely interested in an ES6 rewrite :+1:

**Pull request closed without merging**